app/controllers/events_controller.rb

```diff
@@ -10,6 +10,11 @@ def index
   def show
   end
 
+  def add_me
```

Comment:
This action is not CRUD action. In this task we will need new controller (for example `event_members_controller` or `members_controller` dependent on events) but we can stay with `create` action. You can check [Rails documentation about that](https://guides.rubyonrails.org/routing.html#nested-resources) or we can discuss it together. What you have here will not create a relation between user and event. If you remember we did that only from console now. You can try to similar thing here.